cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -926,6 +927,15 @@ def run_call_back(self, status_kw_name='action') -> typing.Tuple[str, typing.Uni
                     product_type = original_request_par_dic['product_type']
                 except KeyError as e:
                     raise MissingRequestParameter(repr(e))
+                # get more info regarding the status of the request
+                status_details = None
+                if status == 'done':
+                    # set instrument
+                    self.set_instrument(self.instrument_name)
+                    status_details = self.instrument.get_status_details(par_dic=original_request_par_dic,
```

Comment:
exception in fetching would return something here, yes?
It's true that it's hard to always distinguish from the status_details if it's good.
We would probably need to have some defined structure for the output json. Something that contains {... "status": "empty"/"exception"/"successful" ...} etc

Reply:
> exception in fetching would return something here, yes?
>
`status_details` is `None` in case of no exception, Connection related exception and Dispatcher related exception.

Reply:
now a more detailed informative object is included also in the output json